common/lib/xmodule/xmodule/js/spec/video/video_caption_spec.js

```diff
@@ -223,6 +230,22 @@
                         $('.lang').mouseleave();
                         expect($('.lang')).not.toHaveClass('is-opened');
                     });
+
```

Comment:
I've got @clytwynec as a reviewer of the tests, but can loop in @andy-armstrong if you think we need another.